src/homeWork_7_3/Main.java

```diff
@@ -1,22 +1,26 @@
 package homeWork_7_3;
 
+import java.util.InputMismatchException;
 import java.util.Random;
 import java.util.Scanner;
 
+
 public class Main {
 
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
+        System.out.print("Select array size: ");
 
-        Fish fish = new Fish("Fish", 1);
-        Cat cat = new Cat("Cat", 2);
-        Raven raven = new Raven("Raven", 3);
-        Python python = new Python("Python", 4);
-        Rhinoceros rhinoceros = new Rhinoceros("Rhinoceros", 5);
-        Lama lama = new Lama("Lama", 6);
+        int size = -1;
 
-        System.out.print("Select array size: ");
-        int size = scanner.nextInt();
+        try {
+            size = scanner.nextInt();
+            Animal[] animals = new Animal[size];
+        } catch (InputMismatchException e) {
```

Comment:
You cannot use array of animals in this case. Animal[] animals; before try/catch block and then try to initialize them into try